### PR TITLE
Bump ovs-cni to 0.8.0

### PR DESCRIPTION
**Pull request by qinqon**
To include the bridge reporting fix [1]

[1] https://github.com/kubevirt/ovs-cni/pull/93

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by phoracek** [comment]
@qinqon don't forget to update `test/releases`

**Comment by phoracek** [comment]
The marker also fails with:

```
[phoracek@t470s cluster-network-addons-operator]$ ./cluster/kubectl.sh logs -n cluster-network-addons ovs-cni-amd64-4vqm5 -c ovs-cni-marker
F1128 19:01:51.701489       1 main.go:42] failed to create a new marker object: Error creating the ovsdb connection: failed to connect to ovsdb error: Invalid socket file
```

**Comment by phoracek** [comment]
We gotta update the manifest, it has been changed on ovs-cni in https://github.com/kubevirt/ovs-cni/pull/80.

**Comment by qinqon** [comment]
deploying and activaing OVS works fine.
but CI fails at 
```
F1129 11:03:14.970689       1 main.go:42] failed to create a new marker object: Error creating the ovsdb connection: failed to connect to ovsdb error: Invalid socket file

```

**Comment by phoracek** [comment]
That's what I commented. The difference is in the format of the socket parameter.

**Comment by qinqon** [comment]
Also the path, /run/openvswitch is no good. Now it has all it needs also I think we no longer need localbin since we are not using ovs-vsctl

File: README.md [hunk 1]
```diff
@@ -135,9 +135,7 @@ systemctl start NetworkManager
 ## Open vSwitch
 
 The operator allows administrator to deploy [OVS CNI plugin](https://github.com/kubevirt/ovs-cni/)
-simply by adding `ovs` attribute to `NetworkAddonsConfig`. Please note that
```
Comment (phoracek) [inline README.md]: Please say that openvswitch has to be running on the node.

**Pull request merged**